tests/e2e/config/jest.config.js

```diff
@@ -30,6 +30,11 @@ module.exports = {
 		'<rootDir>/tests/e2e/config/jest.setup.js',
 		'expect-puppeteer',
 	],
+	testMatch: [
+		'**/__tests__/**/*.[jt]s?(x)',
+		'**/?(*.)+(spec|test).[jt]s?(x)',
+		'!**/?(*.)+(utils).[jt]s?(x)',
```

Comment:
The first two lines are the default from the jest configuration (source: https://jestjs.io/docs/28.x/configuration#testmatch-arraystring).
The last one is for not running jest on all the files that have the suffix `.utils` or called `utils`.